src/services/vanilla.js

```diff
@@ -204,13 +204,14 @@ async function createVanillaGroup (challenge) {
       }
 
       logger.info(`Creating Vanilla entities for the '${challengeDetailsDiscussion.name}' discussion ....`)
-
-      const groupNameTemplate = _.template(groupTemplate.group.name)
+      
+      const shortGroupName = groupTemplate.group.name.slice(0,config.FORUM_TITLE_LENGTH_LIMIT);
+      const groupNameTemplate = _.template(shortGroupName)
```

Comment:
This won't work either. You're slicing the template which is probably something like `{{challenge.name}}`. You need to slice the result and not the template which will look something like this:
```
name: groupNameTemplate({ challenge }).slice(0,config.FORUM_TITLE_LENGTH_LIMIT)
```